esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -758,6 +758,11 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
 
         implType = getEsdlMethodImplType(tgtcfg->queryProp("@querytype"));
 
+        bool use_numeric_bool = srvdef.getPropBool("numeric_bool");
+        unsigned txResultFlags = ESDL_TRANS_OUTPUT_ROOT;
+        if (use_numeric_bool)
+            txResultFlags |= ESDL_TRANS_NUMERIC_BOOLEAN;
+
```

Comment:
So the service level setting is applied here, but is possible to override in the method when calling process or processHPCCResult.

Reply:
yes, exactly.  The method override exists int the common transformer code.. but setting the default from the service level info had to happen here.